android/assets/jsons/Civ V - Vanilla/TileImprovements.json

```diff
@@ -101,14 +101,20 @@
 		"turnsToBuild": 4,
 		"techRequired": "The Wheel",
 		"uniques": ["Can be built outside your borders", "Costs [1] gold per turn when in your territory"],
-		"shortcutKey": "R"
+		"shortcutKey": "R",
+        "civilopediaText": [
+            {text:"Reduces movement cost to ½ if the other tile also has a Road or Railroad"},
+            {text:"Reduces movement cost to ⅓ with Machinery",link:"Technology/Machinery"},
```

Comment:
These are dependent on tech uniques, and so shouldn't be part of the civilopedia text, methinks